### PR TITLE
doc/glossary: add "ceph-fuse" entry

### PR DESCRIPTION
**Pull request by zdover23**
Add a "ceph-fuse" entry to the glossary.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/glossary.rst [hunk 1]
```diff
@@ -90,6 +90,11 @@
                 object store, RADOS.  See :ref:`CephFS Architecture
                 <arch-cephfs>` for more details.
 
+        :ref:`ceph-fuse <man-ceph-fuse>`
+                :ref:`ceph-fuse <man-ceph-fuse>` is a FUSE ("**F**\ilesystem in
+                **USE**\rspace") client for CephFS. ceph-fuse mounts a Ceph FS
+                ata  specified mount point.
```
Comment (anthonyeleven) [inline doc/glossary.rst:96]: on a specified mountpoint

**Review by anthonyeleven — CHANGES_REQUESTED** [review]

**Review by anthonyeleven — APPROVED** [review]

File: doc/glossary.rst (hunk 1 above)
Comment (zdover23) [reply doc/glossary.rst:96]: Check this out:

```
zdover@NUC8i7BEH:~/Documents/ceph/doc$ git grep -c "mountpoint" *
cephfs/ceph-dokan.rst:3
cephfs/client-auth.rst:1
cephfs/client-config-ref.rst:1
cephfs/mount-using-fuse.rst:2
cephfs/mount-using-kernel-driver.rst:1
dev/freebsd.rst:1
man/8/ceph-authtool.rst:1
man/8/ceph-fuse.rst:3
man/8/ceph-objectstore-tool.rst:2
man/8/mount.ceph.rst:1
man/8/mount.fuse.ceph.rst:1
man/8/rbd-fuse.rst:3
rados/configuration/common.rst:1
rados/operations/user-management.rst:1
zdover@NUC8i7BEH:~/Documents/ceph/doc$ git grep -c "mount point" *
ceph-volume/lvm/prepare.rst:1
ceph-volume/lvm/zap.rst:1
cephfs/cephfs-mirroring.rst:1
changelog/v0.56.5.txt:1
changelog/v0.94.8.txt:1
changelog/v10.2.3.txt:1
changelog/v10.2.6.txt:1
changelog/v10.2.8.txt:2
changelog/v11.2.1.txt:2
man/8/ceph-fuse.rst:1
man/8/mount.ceph.rst:1
radosgw/nfs.rst:1
releases/hammer.rst:3
releases/jewel.rst:3
releases/kraken.rst:1
releases/luminous.rst:2
```

Comment (zdover23) [reply doc/glossary.rst:96]: I'm not taking a side on which one is right at this time, but I am pointing out that whatever we choose to do, we've got some work cut out for ourselves.

Comment (zdover23) [reply doc/glossary.rst:96]: The IBM Style Guide differs with @anthonyeleven: "**mount point** _n_ Write as shown - two words." (IBM Style Guide, p. 345)

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/58943 - Squid backport
https://github.com/ceph/ceph/pull/58944 - Reef backport
https://github.com/ceph/ceph/pull/58945 - Quincy backport

File: doc/glossary.rst (hunk 1 above)
Comment (anthonyeleven) [reply doc/glossary.rst:96]: Feh, the conference in 2018 was `Mountpoint` ;)

**Pull request merged**